org.eclipse.dartboard/plugin.xml

```diff
@@ -174,7 +174,7 @@
        <launchGroup
              bannerImage="platform:/plugin/org.eclipse.debug.ui/icons/full/wizban/run_wiz.png"
```

Comment:
We need to update the appropriate image if available

Reply:
Done. I added a 64x64 icon and referenced it here. Do you know where I can see the result? I couldn't find out where this icon is used.

Reply:
Nevermind, found the location on the Dialog opened, when trying to run a file as Dart program, that has no launch config yet.